libs/features/georoutingv2/georoutingv2.js

```diff
@@ -257,7 +257,7 @@ async function showModal(details) {
   const promises = [
     tabs ? loadBlock(tabs) : null,
     tabs ? loadStyle(`${miloLibs || codeRoot}/blocks/section-metadata/section-metadata.css`) : null,
-    loadStyle(`${miloLibs || codeRoot}/features/georoutingv2/georoutingv2.css`),
+    new Promise((resolve) => { loadStyle(`${miloLibs || codeRoot}/features/georoutingv2/georoutingv2.css`, resolve); }),
   ];
   await Promise.all(promises);
   // eslint-disable-next-line import/no-cycle
```

Comment:
Can/should we also move L264 importing `modal.js` into the promise.all?

Reply:
Hi Okan, I've created a refactor ticket to look into this. [MWPW-152228](https://jira.corp.adobe.com/browse/MWPW-152228)